hugo/content/info/credits.md

```diff
@@ -3,10 +3,38 @@ Title: Credits
 menu: footer
 ---
 
-The staff of the ACL Anthology would like to acknowledge all of the contributors
-and institutions that make this collection of scholarly papers freely available
-to all. The Anthology is only maintained by volunteers and the community's help
-is what continues to make this resource the landmark that it is.
+The staff of the ACL Anthology would like to acknowledge all of the contributors and institutions that make this collection of scholarly papers freely available to all.
+The Anthology is only maintained by volunteers and the community's help is what continues to make this resource the landmark that it is.
+
+### Volunteers
```

Comment:
I think our most dedicated @davidweichiang should appear somewhere here...? :)

Reply:
Yes, what an oversight!